src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -271,78 +272,88 @@ export const CollectCustomerDetailsScreen: FunctionComponent<
 
   const tCampaignName = c13nt(features?.campaignName ?? "");
 
+  const [shouldBlock, setShouldBlock] = useState(true);
+  useEffect(() => {
```

Comment:
block for the first time when user navigates to `CollectCustomerDetailsScreen`

https://github.com/rationally-app/mobile-application/assets/43963814/c24788d9-6bb5-4767-bc50-489633e90e5f